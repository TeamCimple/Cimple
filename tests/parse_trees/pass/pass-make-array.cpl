int main() {
	int *arg;
	
	arg = make int[4];
	clean arg;
	return 0;
}
