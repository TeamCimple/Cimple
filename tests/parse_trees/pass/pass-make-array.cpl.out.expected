FuncDecl(Name: Var(main) ReturnType: int Parameters:  Body: CompoundStatement(Declarations: (int Pointer(Var(arg)),),  StatementList: Statement(EQUALS(Identifier(arg), Make([int]))), Statement(Clean(Identifier(arg))), RETURN(Int(0)))) 
